docker/config_template.yaml

```diff
@@ -64,6 +64,8 @@ persistence:
                 maxConns: {{ default .Env.SQL_MAX_CONNS "20" }}
                 maxIdleConns: {{ default .Env.SQL_MAX_IDLE_CONNS "20" }}
                 maxConnLifetime: {{ default .Env.SQL_MAX_CONN_TIME "1h" }}
+                tls:
+                    enabled: {{ default .Env.SQL_TLS_ENABLED "false" }}
```

Comment:
can you also add `caFile`, `certFile`, `keyFile` & `enableHostVerification`?
ref:
https://github.com/temporalio/temporal/blob/master/docker/config_template.yaml#L26-LL28
https://github.com/temporalio/temporal/blob/master/docker/config_template.yaml#L32

Reply:
Done. I've also added the ability to specify [TLS.ServerName](https://github.com/temporalio/temporal/blob/30c89b9f91dafc13d2d75fda266d81ce8bc72f3e/common/auth/tls.go#L44) through `{CASSANDRA,SQL}_HOST_NAME` env vars.